src/pip/_internal/resolution/resolvelib/factory.py

```diff
@@ -98,9 +98,16 @@ def __init__(
         self._editable_candidate_cache = {}  # type: Cache[EditableCandidate]
 
         if not ignore_installed:
+            packages = get_installed_distributions(
+                local_only=False,
+                skip=(),
```

Comment:
Probably don’t want to do this. The `skip` packages are there because they cause edge cases in the interpreter. This is one of the rare cases I would prefer using the argument defaults if possible instead of being explicit.

Reply:
done